wolox-books/src/app/store/books.reducer.ts

```diff
@@ -0,0 +1,18 @@
+import { Action } from '@ngrx/store'
+import { Book, ShoppingCarState } from '../models/lists-books.model'
+import * as ListBooksCarActions from './books.actions'
+
+const initialState: ShoppingCarState = { books: [], loading: false}
+
+export function reducer(state = initialState, action: ListBooksCarActions.Actions) {
+    switch(action.type) {
+        case ListBooksCarActions.ADD_BOOKS:
+            return {...state, books: [...state.books, action.payload]}
+        
+        case ListBooksCarActions.REMOVE_BOOKS:
+            return [...state.books, action.payload]
```

Comment:
is this working?, what's supposed to be returned by the reducer? an array of books?